code/src/GridAPI/Sort.ts

```diff
@@ -25,4 +25,24 @@ namespace GridAPI.Sort {
             'Sort.Clear-end'
         );
     }
+
+    export function ColumnSort(
+        gridID: string,
+        columnID: string,
+        sorting: OSFramework.OSStructure.Sorting
+    ): void {
+        PerformanceAPI.SetMark('Sort.ColumnSort');
+
+        if (!OSFramework.Helper.IsGridReady(gridID)) return;
```

Comment:
why is not a true condition? and then encapsulate the following code...

Reply:
We've been using this for all API methods. I just followed the pattern.

Reply:
Changed it